client/state/data-layer/wpcom/me/transactions/test/index.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { createTransaction, createSuccess, createError } from '../';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { TRANSACTION_CREATE_REQUEST, TRANSACTION_CREATE_SUCCESS, TRANSACTION_CREATE_FAILURE } from 'state/action-types';
+
+describe( 'data-layer/me/transactions', () => {
+	describe( '#createTransaction', () => {
+		const action = { type: TRANSACTION_CREATE_REQUEST };
+
+		test( 'should create HTTP request to /me/transactions endpoint', () => {
+			expect( createTransaction( action ) ).to.deep.equal(
```

Comment:
We don't need `chai` or `chai.expect` here. We can just replace `to.deep.equal` with `toEqual` (Jest) and do away with the `import { expect } from 'chai';`. 🍾